lib/manageiq-appliance_console.rb

```diff
@@ -39,6 +39,9 @@ def self.logger=(logger)
 require 'manageiq/appliance_console/external_auth_options'
 require 'manageiq/appliance_console/external_database_configuration'
 require 'manageiq/appliance_console/external_httpd_authentication'
+require 'manageiq/appliance_console/message_configuration'
```

Comment:
Sort alphabetically?

Reply:
Can this be merged into https://github.com/ManageIQ/manageiq-appliance_console/pull/130/files#diff-7a5a900c5a6ca9e861d311795135eaa7e7556074b5dc71399e90e12a7910b946R49 ?

Reply:
@bdunne Sorry that link is not clear. Where are you suggesting I merge this require into?

Reply:
https://github.com/ManageIQ/manageiq-appliance_console/blob/481d9b37c73a44471b13ce5c5d7a8fce00b7429d/lib/manageiq-appliance_console.rb#L49

Reply:
@bdunne Alphabetize requires 👍 As discussed combining messaging_configuration with message_configure_client will be done in the future.